option.go

```diff
@@ -64,6 +70,13 @@ func WithPromptString(s string) Option {
 	}
 }
 
+// WithPromptString changes the prompt string. The default value is "> ".
```

Comment:
This comment is the same as `WithPromptString`.

Reply:
Fixed!